codalab/lib/codalab_manager.py

```diff
@@ -395,9 +395,10 @@ def emailer(self):
                 host=self.config['email']['host'],
                 user=self.config['email'].get('user', 'noreply@codalab.org'),
                 password=self.config['email'].get('password', None),
-                use_tls=True,
-                default_sender='CodaLab <noreply@codalab.org>',
-                server_email='noreply@codalab.org',
+                use_tls=self.config['email'].get('use_tls',True),
+                port=self.config['email'].get('port', 587),
+                default_sender=self.config['email'].get('default_sender','CodaLab <noreply@codalab.org>'),
```

Comment:
Could we change the config keys:
- `default_sender` -> `sender_from`
- `server_email` -> `sender_email`
to make it semantically more clear?

Reply:
Sure. and already done